src/strpak.cpp

```diff
@@ -1327,45 +1332,13 @@ BOOL strMatch(					// string match
 }			// strMatch
 //-----------------------------------------------------------------------------
 #if CSE_COMPILER != CSE_COMPILER_MSVC
-inline int _stricmp(	// Substitude windows _stricmp functions
-	const char* char1,	// First string to be compare
-	const char* char2)	// Second string to be compare
-// Compares two string ignoring case sensitivity
-// Eventually replace this function with POSIX standard
-{
-	int sum{ 0 };
-	for (;; char1++, char2++) {
-		sum += tolower((unsigned char)*char1) - tolower((unsigned char)*char2);
-		if (sum != 0 || *char1 == '\0' || *char2 == '\0') {
-			return sum;
-		}
-	}
-} // _stricmp
-//-----------------------------------------------------------------------------
-inline int _strnicmp(			// Substitude windows _strnicmp
-	const char* char1,	// First string to be compare
-	const char* char2,	// Second string to be compare
-	size_t count)		// Number of characters to compare
-// Compares two string ignoring case sensitivity upto the count.
-// Eventually replace this function with POSIX standard
-{
-	int sum{ 0 };
-	for (size_t i = 0; i < count; i++, char1++, char2++) {
-		sum += tolower((unsigned char)*char1) - tolower((unsigned char)*char2);
-		if (sum != 0 || *char1 == '\0' || *char2 == '\0') {
-			return sum > 0? 1:-1;
-		}
-	}
-	return 0;
-}	// _stricmp
-//-----------------------------------------------------------------------------
 // TODO (MP) -- this works but certainly not elegant
 char* _strupr(char* stringMod) // Substitude strupr function
 // Converts a string to uppercase
 {
 	char* temp = stringMod;
-	for (;*temp;++tmp) {
-		*temp = toupper(static_cast<unsigned char>(*temp))
+	for (;*temp;++temp) {
```

Comment:
How was this even compiling (let alone working) before!?

Reply:
This is why we have a unit test. I don't know.